superset-frontend/src/visualizations/TimeTable/TimeTable.jsx

```diff
@@ -98,12 +98,13 @@ const defaultProps = {
   url: '',
 };
 
+// @z-index-above-dashboard-charts + 1 = 11
```

Comment:
FWIW, this is in the LESS codebase as `z-index-above-dashboard-charts` but obviously we're not using LESS here. This PR seems fine, but I wonder if we should make a levels/z-index registry in our Emotion theme. @michael-s-molina @eric-briscoe we might want to have a chat about how we can clean/support/scale these layers.

Reply:
or review, most of those should not be needed at all